documentation/specs/src/economics/public-goods-funding.md

```diff
@@ -1,45 +1,193 @@
-- Stake-weighted approval voting: as public goods councils are exclusive, we can use a stake-weighted form of approval voting. Governance voters include all public goods council candidates of which they approve, and the council candidate with the most stake approving it wins. This doesn't have game-theoretic properties as nice as ranked-choice voting (especially when votes are public, as they are at the moment), but it is _much_ simpler ([background](https://en.wikipedia.org/wiki/Condorcet_method)), and in practice I do not think there will be too many public goods council candidates.
-- Interface support: the interface should support limited liquid democracy for delegate selection and approval voting for public goods council candidates. The interface or explorer should display past retroactive PGF winners and past/current continuous funding recipients. Proposal submission for continuous and retroactive funding will happen separately, in whatever manner the public goods council deems fit.
+### Constructing the council
+All valid PGF councils will be established multisignature account addresses. These must be created by the intdended parties that wish to create a council. The council will therefore have the discretion to decide what threshold will be required for their multisig (i.e the "k" in the "k out of n").
+
+The council will be resonsible to publish this address to voters and express their desired `spending_cap`. 
+
+A council consisting of the same members should also be able to propose multiple spending caps (with the same multisig address). These will be voted on as separate councils and votes counted separately.
+
+
+### Voting on the council
+Once the council has been constructed, it can be voted on by governance particpants through a vote as structured below:
+
+```rust
+struct OnChainVote {
+    id: u64,
+    voter: Address,
+    yay: proposalVote,
+}
+```
+
+In turn the proposal vote will include the a structure like:
+
+```rust
+HashSet<(Address, u64)>
+```
+
+The structure contains all the counsils voted, where each cousil is specific as a pair `Address` (the enstablished address of the multisig account) and `u64` (spending cap)
+
+These votes will then be used in order to vote for various PGF councils. Multiple councils can be voted on through a vector as represented above.
+
+
+### CLI
+
+TODOs:
+
+- PGF Votes
+- Construction of PGF council multisigs 
+
+### Dealing with ties
+In the rare occurance of a tie, the council with the lower spending_cap will win the tiebreak.
+
+In the case of equal tiebreaks, the addresses with lower alphabetical order will be chosen. This is very arbitrary due to the expected low frequency.
+
+### Ensuring enough votes 
+
+In order for a new PGF council to be elected (and hence halting the previous council's power), $\frac{1}{3}$ of validating power must vote on the "New PGF Council Proposal". Once this condition has been met, majority vote will be decided as to which PGF council will be elected.
```

Comment:
majority? I think you mean the (council, spending_cap) pair with the most votes, right?

this is approval voting (a pretty standard mechanism, so worth using the name)

Reply:
Do we need 1/3 of the voting power to vote on the proposal or to vote `Yay` on the proposal (regardless of the specific councils voted)? I'm thinking about a lot of voting power voting for `Nay` and a very small power voting for a council that ends up being elected

Reply:
we need 1/3 to vote "yay" on the proposal, yeah - then the council with the most votes (which could be under 1/3) gets elected